CSharp/Mazes/Grid/Grid.cs

```diff
@@ -47,6 +52,10 @@ public int GetColumns()
             return columns;
         }
 
+        /*
```

Comment:
Use /// for method comments.

Don't talk about RoomHelper here.  Move the comment to RoomHelper.  Remove the sentence with `starts out as null`.  People can see from the code that roomToConnect with is null and you don't need to explain it again.  

Method level comments should explain what it does, not the how.